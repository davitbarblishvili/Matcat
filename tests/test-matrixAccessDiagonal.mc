func main() int {
    matrix a; 
   
    a = [[1,2,3],[4,5,6],[7,8,9]];
    print_diagonal(a);


    return 0;
}

