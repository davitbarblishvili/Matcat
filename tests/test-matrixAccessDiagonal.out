(
[1.00 5.00 9.00 ]
)